clubs/models.py

```diff
@@ -328,6 +328,7 @@ class Asset(models.Model):
     """
     creator = models.ForeignKey(get_user_model(), null=True, on_delete=models.SET_NULL)
     file = models.FileField(upload_to=get_asset_file_name)
+    club = models.ForeignKey(Club, on_delete=models.CASCADE, null=True)
```

Comment:
You probably don't want `null=True` if you use `models.CASCADE`.

For the `creator` field, this is set to true because when the creator of the file is deleted, you set the `creator` field to null instead of deleting the asset. In the case of clubs, when the club is deleted you probably want to delete the associated assets.